### ISSUE TITLE
Web - Steps Pages - User Can Tab through Closed Menu Items

### ISSUE THREAD

**Issue by jfc3-dol** [USDepartmentofLabor/Comply-Chain#147]
On the Steps pages, at least, when the Menu is closed, the user has to tab through the hidden items in Chrome, Edge, and IE 11. The problem might be on other pages too.

The issues is related to JIRA issue CCP-119.

**Comment by harpreetsdev** [comment]
@jfc3-dol 

This issue has been fixed. You should be able to see the changes in the Sprint 11 build.

@pbhatt17 @Abychalil

**Comment by harpreetsdev** [comment]
@jfc3-dol 

Please remember clearing cache before testing this issue and others. The fix for this issue has worked fine on my machine and our systems engineer machine but is cropping up on our test engineer's machine every once in a while. So your machine is going to be a deciding vote on whether this issue is completely fixed or not.

@pbhatt17 @Abychalil

**Comment by jfc3-dol** [comment]
@harpreetsdev @pbhatt17 @Abychalil

No longer need to tab through the hidden Menu items on the Steps pages or even the home page in IE11, Edge, Firefox, and Chrome.

So I'm going to close this issue.
